pkg/apis/kai/v1/config_types.go

```diff
@@ -0,0 +1,156 @@
+// Copyright 2025 NVIDIA CORPORATION
+// SPDX-License-Identifier: Apache-2.0
+
+package v1
+
+import (
+	"github.com/NVIDIA/KAI-scheduler/pkg/apis/kai/v1/admission"
+	"github.com/NVIDIA/KAI-scheduler/pkg/apis/kai/v1/binder"
+	"github.com/NVIDIA/KAI-scheduler/pkg/apis/kai/v1/node_scale_adjuster"
+	"github.com/NVIDIA/KAI-scheduler/pkg/apis/kai/v1/pod_group_controller"
+	"github.com/NVIDIA/KAI-scheduler/pkg/apis/kai/v1/pod_grouper"
+	"github.com/NVIDIA/KAI-scheduler/pkg/apis/kai/v1/queue_controller"
+	"github.com/NVIDIA/KAI-scheduler/pkg/apis/kai/v1/scheduler"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+type ConditionType string
+
+const (
+	ConditionTypeReconciling       ConditionType = "Reconciling"
+	ConditionTypeDeployed          ConditionType = "Deployed"
+	ConditionTypeAvailable         ConditionType = "Available"
+	ConditionDependenciesFulfilled ConditionType = "DependenciesFulfilled"
+)
+
+const defaultKAINamespace = "kai-scheduler"
+
+type ConditionReason string
+
+const (
+	Deployed              ConditionReason = "deployed"
+	Available             ConditionReason = "available"
+	Reconciled            ConditionReason = "reconciled"
+	DependenciesFulfilled ConditionReason = "dependencies_fulfilled"
+)
+
+// ConfigSpec defines the desired state of Config
+type ConfigSpec struct {
+	// Namespace is the namespace to create the operands in
+	// +kubebuilder:validation:Optional
+	Namespace string `json:"namespace,omitempty"`
+
+	// Global defined global configuration of the system
+	// +kubebuilder:validation:Optional
+	Global *GlobalConfig `json:"global,omitempty"`
+
+	// PodGrouper specifies configuration for the pod-grouper
+	// +kubebuilder:validation:Optional
+	PodGrouper *pod_grouper.PodGrouper `json:"podGrouper,omitempty"`
+
+	// Binder specifies configuration for the binder
+	// +kubebuilder:validation:Optional
+	Binder *binder.Binder `json:"binder,omitempty"`
+
+	// Admission holds kai admission webhooks
```

Comment:
```suggestion
	// Admission specifies configuration kai admission webhooks
```
Just seems to fit the rest better